src/parser/vast_parser.js

```diff
@@ -125,254 +142,279 @@ export class VASTParser extends EventEmitter {
    * @emits    VASTParserr#VAST-resolved
    * @callback cb
    */
-  getAndParse(url, options, cb) {
+  getAndParseVAST(url, options, cb) {
     if (!cb) {
```

Comment:
It means its a function with 2 or 3 params, the last one being the callback?  
I'm not a big fan of this pattern. Though, I haven't thought of a better way to handle it.  
It has nothing to do with the PR, so not a big deal.

Reply:
Yes I don't like it either. I just kept the pattern we had before. One option could be return a `Promise` which resolves with a `VASTResponse` or rejects with an `Error`.